{{cookiecutter.project_slug}}/clients/web/react/src/services/axios-instance.ts

```diff
@@ -9,6 +10,9 @@ export const axiosInstance = axios.create({
   headers: {
     'Content-Type': 'application/json',
   },
+  paramsSerializer: (params) => {
+    return qs.stringify(params, { arrayFormat: 'repeat' })
```

Comment:
hmm I think we had said the backend prefers csv, this is set to repeat?

Reply:
This was the only example I had. We can set it to repeat instead if that's the case

Reply:
I think it would be better but the test I just run real quick seems to indicate that it works either way! (but that was a quick test)

Reply:
Maybe django understands both. Just updated for it to be comma